### ISSUE TITLE
Recaptcha disappears on mobile 

### ISSUE THREAD

**Issue by AndreiMotinga** [ambethia/recaptcha#164]
I've just noticed that it actually doesn't show up on small screens 
Not sure that it's a bug, but I was hoping someone can help me figure out how to keep in it there. 

I don't see any css that I can tweak in order to keep it visible, any ideas?

**Comment by grosser** [comment]
could try with just a browser window and then making it  smaller until it vanishes
then inspect the styles ... I'd guess there is something with a fixed width ...

**Comment by AndreiMotinga** [comment]
Yeah, this is exactly what I did, was selecting different elements and
trying to find what style changes, but couldn't find it. Thought may
be someone already faced it.

**Comment by langesi** [comment]
We have the same issue

**Comment by viveksoundrapandi** [comment]
we too have the same issue. ANy updates here?
